.github/workflows/test-driver-adapters-template.yml

```diff
@@ -55,6 +55,11 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Cache Docker images.
+        uses: ScribeMD/docker-cache@0.5.0
+        with:
+          key: docker-${{ inputs.setup_task }}-${{hashFiles('docker-compose.yaml')}}
```

Comment:
Why can't we share the cache between the jobs (i.e. `key: docker-${{ hashFiles('docker-compose.yaml') }}`)?

Reply:
Our different jobs pull different docker images. E.g. one might only pull the `mongodb` image, another only the `postgres` one. Would they share the cache each cache would only contain one docker image but cause a cache miss for all others.

Even if we could somehow put all images into one cache we'd cause larger cache download times because it needs to load a cache with all our images instead of just one. With ~200+MB per image and our number of images it would be quite a size I think.

Reply:
got it, makes sense!